src/index.js

```diff
@@ -8,4 +8,5 @@ export { default as LayoutHOC } from './layout.hoc'
 export { version } from '../version'
 export const utils = srcUtils
 
+export { default as Col } from './derived/col'
 export { default as Root } from './derived/root'
```

Comment:
I don't have a problem keeping them top level, since they're going to be very common primitives.